Automattic/wc-calypso-bridge
# Revert the "Blog Home" template name change in the Site Editor to "Home"

### Changes proposed in this Pull Request:

Changes the "Blog Home" template name in the Site Editor back to "Home".

Closes #1262






### How to test the changes in this Pull Request:


On a fresh Woo Express dev site: 

1. Enable Tsubaki.
2. Go to **Appearance > Editor > Templates**.
3. Double check that the home page template is called "Blog Home".

With the changes in this PR enabled:
1. Go to **Appearance > Editor > Templates**.
2. The home page template should now be called "Home".




### Other information:




### FOR PR REVIEWER ONLY:
